src/main.cpp

```diff
@@ -20,7 +20,7 @@ int main(int argc, char **argv)
 	render_objects.push_back(&cylinder);
 	render_objects.push_back(&sphere);
 
-
+// kox PL
```

Comment:
Wow. 
Thanks for contributing to my open source project. this is great addition to my vision.